pkg/models/temp_user.go

```diff
@@ -24,22 +26,20 @@ const (
 
 // TempUser holds data for org invites and unconfirmed sign ups
 type TempUser struct {
-	Id              int64
-	OrgId           int64
-	Version         int
-	Email           string
-	Name            string
-	Role            org.RoleType
-	InvitedByUserId int64
-	Status          TempUserStatus
-
-	EmailSent   bool
-	EmailSentOn time.Time
-	Code        string
-	RemoteAddr  string
-
-	Created int64
-	Updated int64
+	Id              int64          `db:"id"`
```

Comment:
I think, eventually, we would like to move these models in the service so it would be good to add a TODO (as a reminder).